vite.config.ts

```diff
@@ -1,22 +1,23 @@
 import path from 'path';
 
+import { vanillaExtractPlugin } from '@vanilla-extract/vite-plugin';
 import solid from 'solid-start/vite';
 import { defineConfig } from 'vite';
 
 export default defineConfig({
-  plugins: [solid()],
+  plugins: [solid(), vanillaExtractPlugin()],
   resolve: {
     alias: {
       '@cubeartisan/cubeartisan': path.resolve(__dirname, './src/'),
     },
   },
-  test: {
-    deps: {
-      registerNodeLoader: true,
-    },
-    globals: true,
-    isolate: true,
-    setupFiles: ['node_modules/@testing-library/jest-dom/extend-expect'],
-    transformMode: { web: [/\.[jt]sx$/] },
-  },
+  // test: {
```

Comment:
Uncomment this and change the import above to import from `'vitest/config' `

Reply:
fixed and pushed